workplace/Living in Open Workspaces.md

```diff
@@ -0,0 +1,80 @@
+## Living in Open Workspaces
+
+Total time: 35 minutes
+
+- Lesson: 15 min
+- Check for Understanding: 20 min
+
+## Motivation
+
+In a [recent study](https://www.business2community.com/human-resources/how-to-design-a-workspace-that-boosts-employee-motivation-02222288), research company IPSOS, discovered that a workspace focused on employee happiness is a powerful driver for productivity.
+
+The IPSOS study showcases the urgent need for more engaging workspaces:
+
+- 85% of workers feel they don’t work in a space where they can share ideas
+- 87% feel there isn’t a place to work in teams
+- 84% don’t feel a sense of belonging or a willingness to be a part of company culture
+
```

Comment:
This is cool.  Can you explain what this has to do with open workspaces?

Reply:
I do not understand. Do you want me to remove this section or add anything? If you want to append anything, please suggest what to add.

Reply:
There is no context here. Tell them what this has to do with **open** workspaces.